src/common/ignore.ts

```diff
@@ -57,3 +58,31 @@ export function getRuleIgnoreParams(node: Model | Enum, ruleName: string) {
   const params = ruleIgnoreComment.split(' ').slice(-1)[0];
   return params.split(',').map((p: string) => p.trim());
 }
+
+export function isFieldIgnored(
```

Comment:
please add some unit tests for this function in particular

Reply:
There are now few edge-case tests for the comment and few new tests for rules